src/validators/admin.validator.ts

```diff
@@ -0,0 +1,48 @@
+import { NextFunction } from 'express';
+import { IRequest, IResponse } from '../interfaces/vendor';
+import { validator } from '../utils';
+import * as regexes from '../utils/constants/regexes';
+import { ValidationException } from '../utils/exceptions';
+
+const validateAdminRegistration = (req: IRequest, res: IResponse, next: NextFunction) => {
```

Comment:
use INextFunction ?

Reply:
Planning to remove INextFunction in next iteration

Reply:
Done removed